app/assets/javascripts/discourse/app/components/emoji-picker.js

```diff
@@ -325,8 +317,10 @@ export default Component.extend({
         const emojiBelow = [...emojis]
           .filter((c) => c.offsetTop > active.offsetTop)
           .find((c) => c.offsetLeft === active.offsetLeft);
-        this._updateEmojiPreview(emojiBelow.title);
-        emojiBelow?.focus();
+        if (emojiBelow) {
+          this._updateEmojiPreview(emojiBelow.title);
```

Comment:
This potentially fixes a bug since there could be a case where `emojiBelow` could be `null`, so calling `.title` on it would throw an exception.